### ISSUE TITLE
Download not working

### ISSUE THREAD

**Issue by khalilhoti** [MattJAshworth/TikSave#1]
when link copied it show downloading toast but not downloading

**Comment by MattJAshworth** [comment]
Yeah I'm aware of this issue, I'm not actively working on this project right now so cannot provide a fix estimate. You're more than welcome to try yourself though.
